features/step_definitions/back_office/user_roles_steps.rb

```diff
@@ -71,3 +71,11 @@
   @world.bo.dashboard_page.submit(search_term: "Mr Waste")
   expect(@world.bo.dashboard_page.resume_links.count).to eq(0)
 end
+
+Then("I can access data exports") do
+  @world.bo.dashboard_page.load
+  expect(@world.bo.dashboard_page.admin_menu).to have_data_exports
+
+  @world.bo.dashboard_page.admin_menu.data_exports.click
+  expect(find("h1.heading-large").text).to eq("Data Exports")
```

Comment:
My first thought here was why isn't the heading in a page object? But if we're pretty sure that we'll only use this once, I'm happy that this option results in less code.